app/nexus/internal/env/backend.go

```diff
@@ -45,6 +51,6 @@ func BackendStoreType() StoreType {
 	case string(Memory):
 		return Memory
 	default:
-		return Memory
+		return Sqlite
```

Comment:
Defaulting to SQLite.

We'll have a `--dev` flag that will disable keeper interaction and use an in-memory backing store (later down the line).